Week3Ses1/Week3Ses1/Program.cs

```diff
@@ -0,0 +1,205 @@
+﻿// See https://aka.ms/new-console-template for more information
+using System;
+using System.Linq.Expressions;
+
+Console.WriteLine("-------Ex1---------");
+int[] numbers = new int[]{1, 3, 2, 6, 12};
+int sum = 0;
+for (int i = 0; i < numbers.Length; i++)
+{
+    if (numbers[i] %2 ==0) { sum = sum+ numbers[i];}
+}
+Console.WriteLine("Suma nr pare este " + sum);
+
+Console.WriteLine("--------Ex2---------");
+int[] numb = new int[] { 6, 3, 4, 3, 6 };
+for (int i = 0; i <= (numbers.Length-1); i++)
+{
+    
+   for (int j = i+1; j < numbers.Length; j++)
+    if (numb[i]  == numb[j]) { Console.WriteLine(numb[i]); }
+}
+
+
+Console.WriteLine("-------Ex3------");
+var litere = new List<string>() { "a", "b", "b", "c", "c", "d" };
+for (int i = 1;i<=litere.Count-1;i++)
+{
+    if (litere[i-1] == litere[i])
+    {
+        litere.RemoveAt(i);
+        i--;
+    }
+}
+for (int i = 0; i < litere.Count; i++)
+{
+    Console.WriteLine(litere[i]);
+}
+
+Console.WriteLine("-------Ex4------");
+
+int[] newnumb = new int[] { 1, 4, 5, 2, 1, 4, 3, 1, 2 };
+for (int i = 0; i < newnumb.Length; i++)
+{
+    int count = 1;
+    for (int j = i + 1; j < newnumb.Length; j++)
+    {
+        if (newnumb[i] == newnumb[j])
+        {
+           
+            count++;
+        }
+    }
+    Console.WriteLine("Numarul " +newnumb[i] + " apare de " + count + " ori");
```

Comment:
This line of code will run too many times. Instead of having in the end one line in he console saying `number x appears 5 times` you will have more lines saying `number x appears 2 times` and then` number x appears 3 times` and so on.